api/config/v1alpha1/envoygateway_types.go

```diff
@@ -40,6 +40,14 @@ type EnvoyGatewaySpec struct {
 	//
 	// +optional
 	Provider *Provider `json:"provider,omitempty"`
+
+	// RateLimit defines the configuration associated with the Rate Limit service
+	// deployed by Envoy Gateway required to implement the Global Rate limiting
+	// functionality. The specific rate limit service used here is the reference
+	// implementation in Envoy. For more details visit https://github.com/envoyproxy/ratelimit.
+	//
```

Comment:
Although the godocs state "Global Rate Limiting", consider adding a line that states this does not apply for "local" rate limiting and reference https://github.com/envoyproxy/gateway/blob/main/api/v1alpha1/ratelimitfilter_types.go for additional details.